BookingSpecTests/SampleTest.feature

```diff
@@ -3,9 +3,122 @@
+	And I write email on Sign in PopUp "testmirantistest@gmail.com"
+	And I write password "1234qweR"
+	And I click Sign In button on PopUp
+	Then I see that i am Signed In
+
+Scenario: Input in email field more than 80 chars and assert that there only 80 chars
+	Given I open browser
+	And I navigate to url "https://booking.com"
+	When I click Sign In button
+	And I write email with length ninety chars "testmirantistesttestmirantistesttestmirantistesttestmirantistesttestmirantistest@gmail.ru"
+	Then I check that length of email is 80 chars
+
+Scenario: Attempt to recover password with invalid email
+	Given I open browser
+	And I navigate to url "https://booking.com"
+	When I click Sign In button
+	When I click Forgot Your Password button
+	And I write email "estmirantistest@gmail.com"
+	And I Click button Send
+	Then I wait while page popup is loading
+	Then I see an error on ForgotYourPasswordPopUp
+
+Scenario: Cancel recover password and Sign in
+	Given I open browser
+	And I navigate to url "https://booking.com"
+	When I click Sign In button
+	When I click Forgot Your Password button
+	And I write email "testmirantistest@gmail.com"
+	Then I click button Cancel
+	When I set following parameters on Sign In Pop Up dialog
+		| Field | Value |
+		| Email | testmirantistest@gmail.com |
+		| Password  | 1234qweR |
+	And I click Sign In button on PopUp
+	Then I see that i am Signed In
+
+Scenario: Click on recover password and Sign In with old password
+	Given I open browser
+	And I navigate to url "https://booking.com"
+	When I click Sign In button
+	When I click Forgot Your Password button
+	And I write email "testmirantistest@gmail.com"
+	And I Click button Send
+	And I wait while page popup is loading
+	Then I click button Back to sign in
+	When I set following parameters on Sign In Pop Up dialog
+		| Field | Value |
+		| Email | testmirantistest@gmail.com |
+		| Password  | 1234qweR |
+	And I click Sign In button on PopUp
+	Then I see that i am Signed In 
+
+Scenario: Input Valid Email and Pass close PopUp after that open PopUp and Sign In
+	Given I open browser
+	And I navigate to url "https://booking.com"
+	When I click Sign In button
+	When I set following parameters on Sign In Pop Up dialog
+		| Field | Value |
+		| Email | testmirantistest@gmail.com |
+		| Password  | 1234qweR |
+	Then I click on X button
+	And  I click Sign In button
+	When I click Sign In button on PopUp
+	Then I see that i am Signed In 
+
+Scenario: Input Valid Email and Pass then Refresh and attempt to Sign In
+	Given I open browser
+	And I navigate to url "https://booking.com"
+	When I click Sign In button
+	When I set following parameters on Sign In Pop Up dialog
+		| Field | Value |
+		| Email | testmirantistest@gmail.com |
+		| Password  | 1234qwER |
+	Then I click on X button
+	When I refresh page
+	And  I click Sign In button
+	When I click Sign In button on PopUp
+	Then I see an error on Sign In PopUp
+
+#Scenario: Recover Password by use link from email
```

Comment:
Let's add a comment why this scenario was commented.
BTW, it's better to use @Ignore tag for such cases.